import-service/src/handlers/importFileParser.ts

```diff
@@ -29,8 +30,15 @@ export const importFileParser = async event => {
           .on('open', () => {
             console.log(`Parsing file ${key}`);
           })
-          .on('data', data => {
+          .on('data', async data => {
             console.log('Parsed data:', data);
+            await sqs.sendMessage(
```

Comment:
if you want to use `await` you should promisify `sendMessage`.
You can use it like this
```
.on('data', (data) => {
    sqs.sendMessage(your params, your callback);
})
```

or

```
.on('data', async (data) => {
    await sqs.sendMessage(your params).promise();;
})
```

Reply:
Thank you a lot for your comment
Due to lack of time will try to correct it during task 7